### PR TITLE
Fix small issues

### PR DESCRIPTION
**Pull request by achamseddine**
@jamescw 

Result display name
Approve trip by supervisor
Do not send trip notification to PA if no budget owner

### CODE REVIEW

File: EquiTrack/trips/forms.py [hunk 1]
```diff
@@ -188,8 +188,9 @@ def clean(self):
         new_locations = sorted([f.instance for f in self.forms if f.cleaned_data])
         if self.instance.approved_by_supervisor:
             old_locations = [i for i in self.instance.triplocation_set.all()]
-            if new_locations != old_locations:
-                raise ValidationError('You cannot modify the location after the trip has been approved')
+            for location in new_locations:
+                if location not in old_locations:
```
Comment (robertavram) [inline EquiTrack/trips/forms.py]: this will allow you to remove locations after the trip was approved.

File: EquiTrack/trips/models.py [hunk 2]
```diff
@@ -402,7 +402,7 @@ def send_trip_request(cls, sender, instance, created, **kwargs):
                     instance.travel_assistant.email
                 )
 
-            if instance.ta_required and instance.programme_assistant and not instance.ta_drafted:
+            if instance.ta_required and instance.programme_assistant and instance.budget_owner and not instance.ta_drafted:
                 emails.TripTAEmail(instance).send(
```
Comment (robertavram) [inline EquiTrack/trips/models.py]: Why do we need to add budget_owner here?

File: EquiTrack/trips/forms.py (hunk 1 above)
Comment (achamseddine) [inline EquiTrack/trips/forms.py]: You alright, I will add a length validation

File: EquiTrack/trips/models.py (hunk 2 above)
Comment (achamseddine) [inline EquiTrack/trips/models.py]: Supply section requested that no needs to send email notification to "Programme assistant" if there's no "Budget owner" selected

**Comment by robertavram** [comment]
Closing and pulling in develop

**Pull request closed without merging**